blazor/dropdown-list/style.md

```diff
@@ -7,10 +7,66 @@ control: DropDown List
 documentation: ug
 ---
 
-# Style and Appearance in Blazor DropDown List Component
+# Style and Appearance in Dropdown List
 
 The following content provides the exact CSS structure that can be used to modify the control's appearance based on the user preference.
 
+## Read-only mode
+
+You can specify the boolean value to [Readonly](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Readonly) whether the DropDownList allows the user to change the value.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/readonly-mode.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Readonly mode](./images/style/blazor_dropdown_readonly-mode.png)
+
+## Disabled state
+
+You can specifies the boolean value to [Enabled](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_Enabled) property that indicates whether the component is enabled or not.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/disabled-state.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Disabled ](./images/style/blazor_dropdown_disabled-state.png)
+
+## Customizing the disabled component’s text color
+
+Use the following CSS to customize the text color when the component is disabled.
+
+```css
+.e-input-group.e-control-wrapper .e-input[disabled] {
+    -webkit-text-fill-color: #0d9133;
+}
+```
+
+## Show the custom icon in dropdown icon
+
+You can change the dropdown [icon](https://ej2.syncfusion.com/documentation/appearance/icons/#material) by overriding the below style “content”.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/dropdown-icon.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with dropdown icon](./images/style/blazor_dropdown_dropdown-icon.png)
+
+You can also change the dropdown icon for the particular component using [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property and add style to the custom class which is mapped to `CssClass`.
```

Comment:
Change the dropdown icon for the particular component using the [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property and add style to the custom class, which is mapped to the `CssClass.`